include/mbedtls/mbedtls_config.h

```diff
@@ -2800,6 +2800,23 @@
  */
 #define MBEDTLS_GCM_C
 
+/**
+ * \def MBEDTLS_GCM_LARGETABLE
+ *
+ * Enable large pre-computed tables for  Galois/Counter Mode (GCM).
+ * Can significantly increase throughput on systems without GCM hardware
+ * acceleration (e.g., AESNI, AESCE).
+ *
+ * The mbedtls_gcm_context size will increase by 3840 bytes.
+ * The code size will increase by roughly 344 bytes.
+ *
+ * Module:  library/gcm.c
+ *
+ * Requires: MBEDTLS_GCM_C
+ *
```

Comment:
(v v minor) most comment doc blocks don't have an empty comment line at the end

Reply:
Done.